src/App.js

```diff
@@ -91,6 +77,25 @@ class App extends Component {
       </React.Fragment>
     );
   }
+
+  componentDidMount() {
+    getMovies()
+      .then((response) => this.setState({ movies: response.movies }))
+      .then(this.setState({ loader: false }))
+      .catch((err) => this.setState({ error: err }));
```

Comment:
Be sure that "err" is a string as the initial state indicates, and not an object with an error message inside it.